src/uu/chcon/src/chcon.rs

```diff
@@ -155,20 +154,14 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 }
 
 pub fn uu_app() -> Command {
-    Command::new(uucore::util_name())
+    let cmd = Command::new(uucore::util_name())
         .version(uucore::crate_version!())
-        .help_template(uucore::localized_help_template(uucore::util_name()))
         .about(translate!("chcon-about"))
         .override_usage(format_usage(&translate!("chcon-usage")))
-        .infer_long_args(true)
-        .disable_help_flag(true)
+        .infer_long_args(true);
+    uucore::clap_localization::configure_localized_command(cmd)
```

Comment:
What's the reason for splitting the setup of command into two parts? Why not doing the localization at the end?